src/ConfigManager.cpp

```diff
@@ -431,7 +431,14 @@ void ConfigManager::createBaseWebServer() {
 void ConfigManager::streamFile(const char* file, const char mime[]) {
   SPIFFS.begin();
 
-  File f = SPIFFS.open(file, "r");
+  File f;
+  if ((String) file[0] == "/") {
+    f = SPIFFS.open(file, "r");
+  } else {
+    String filepath = "/" + (String) file;
+    f = SPIFFS.open(filepath.c_str(), "r");
+  }
+
```

Comment:
```suggestion
  if (file[0] != '/') {
    size_t len = strlen(file);
    char* path = new char[len+1]
    strcpy (path,"/");
    strcat (path,file);
    file = path
  }

  File f = SPIFFS.open(file, "r");
```
Does this not do the job? A little better than the string conversion IMO.